docs/configuration/index.md

```diff
@@ -30,7 +30,13 @@ This information can be found on the newly created AD Application as documented
 
 The entity in the Azure AD needs to have `Monitoring Reader` permission on the resource group that will be queried. More information can be found [here](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-roles-permissions-security).
 
-# Telemetry
+# Logging & Telemetry
+We provide insights in how our runtime is doing and is written to `stdout`.
+
+This can be controlled via the following environment variables:
+- **PROMITOR_LOGGING_MINIMUMLOGLEVEL** - Defines the minimum log level that should be logged. If none is configured, `Warning` will be used. Allowed values are `Trace`, `Debug`, `Information`, `Warning`, `Error`, `Critical`, `None` ordered from most to least verbose.
```

Comment:
@lee0c I presume this needs to be added to our Helm chart? If so, I'd like to give it a try to learn how it works.

Reply:
Looks like I'll have to do that!